app/services/request_password_reset.rb

```diff
@@ -16,14 +17,22 @@ def perform
   private
 
   def send_reset_password_instructions
+    irs_params = {
+      email: email,
+    }
+
     if Throttle.new(user: user, throttle_type: :reset_password_email).throttled_else_increment?
       analytics.throttler_rate_limit_triggered(throttle_type: :reset_password_email)
+      irs_attempts_api_tracker.forgot_password_email_rate_limited(**irs_params)
```

Comment:
same comment as before, I'd ditch the indirection of `irs_params` variable and just inline this
```suggestion
      irs_attempts_api_tracker.forgot_password_email_rate_limited(email: email)
```